service/provider/src/main/java/gov/va/vro/service/provider/camel/MasIntegrationRoutes.java

```diff
@@ -99,10 +110,22 @@ public void configure() {
   }
 
   private void configureAutomatedClaim() {
+    from(ENDPOINT_REQUEST_INJECTION)
+        .setExchangePattern(ExchangePattern.InOnly)
+        .routeId("mas-request-injection")
+        // Remove the CamelRabbitmqExchangeName and CamelRabbitmqRoutingKey headers so they don't
+        // interfere with future sending to rabbitmq endpoints
+        // https://camel.apache.org/components/3.19.x/rabbitmq-component.html#_troubleshooting_headers:
+        // > if the source queue has a routing key set in the headers, it will pass down
+        // > to the destination and not be overriden with the URI query parameters.
+        .removeHeaders("CamelRabbitmq*")
+        .convertBodyTo(MasAutomatedClaimPayload.class)
+        .wireTap(VroCamelUtils.wiretapProducer(MAS_CLAIM_WIRETAP))
+        .to(ENDPOINT_AUTOMATED_CLAIM);
+
     var checkClaimRouteId = "mas-claim-notification";
     from(ENDPOINT_AUTOMATED_CLAIM)
         .routeId(checkClaimRouteId)
-        .wireTap(VroCamelUtils.wiretapProducer(MAS_CLAIM_WIRETAP))
```

Comment:
This wireTap is for new claims, so move it to the new injection route since this `ENDPOINT_AUTOMATED_CLAIM` route is used as a retry route multiple times for the same claim.